libs/features/mas/web-components/src/merch-datasource.js

```diff
@@ -2,6 +2,7 @@ import { AEM } from './aem.js';
 import { createTag } from './utils.js';
 
 const ATTR_AEM_BUCKET = 'aem-bucket';
+const AEM_BUCKET = 'publish-p22655-e155390';
```

Comment:
I think we should delete the default value and discuss how to pass it as a parameter in a clean way

Reply:
yes, agree here, i just didn't want to bloat this PR yet.
same as @npeltier mentioned above, we need mechanism to switch from prod to qa/stage etc